### PR TITLE
Fpet 647 cfvenhancements fordemo

### PR DESCRIPTION
**Pull request by remya-james**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] the `keep-helm` label has been added, if the helm release should be persisted after a successful build

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: definitions/private-law/json/AuthorisationCaseField/CaseFileView/AuthorisationCaseField.json [hunk 1]
```diff
@@ -93,35 +93,35 @@
   {
     "LiveFrom": "24/11/2022",
     "CaseTypeID": "PRLAPPS",
-    "CaseFieldID": "ordersSubmittedWithApplication",
+    "CaseFieldID": "draftOrders",
```
Comment (priteshkr) [inline definitions/private-law/json/AuthorisationCaseField/CaseFileView/AuthorisationCaseField.json:96]: Please keep the "ordersSubmittedWithApplication" , casefield, Reason being any case who has got this casefield in the casedata will throw the CaseValidationException , if it is removed away .

File: definitions/private-law/json/AuthorisationCaseField/CaseFileView/AuthorisationCaseField.json [hunk 2]
```diff
@@ -160,41 +160,6 @@
     "UserRole": "ctsc-team-leader",
     "CRUD": "CRU"
   },
-  {
-    "LiveFrom": "24/11/2022",
-    "CaseTypeID": "PRLAPPS",
-    "CaseFieldID": "standardDirectionsOrder",
-    "UserRole": "caseworker-privatelaw-courtadmin",
-    "CRUD": "CRU"
-  },
```
Comment (priteshkr) [inline definitions/private-law/json/AuthorisationCaseField/CaseFileView/AuthorisationCaseField.json:169]: please keep this casefield

File: definitions/private-law/json/CaseField/CaseField.json [hunk 3]
```diff
@@ -129,7 +129,7 @@
     "FieldTypeParameter": "Document",
     "SecurityClassification": "Public",
     "Searchable": "N",
-    "CategoryID": "ordersSubmittedWithApplication"
```
Comment (priteshkr) [inline definitions/private-law/json/CaseField/CaseField.json:132]: Please keep this caseField "ordersSubmittedWithApplication"

File: definitions/private-law/json/AuthorisationCaseField/CaseFileView/AuthorisationCaseField.json (hunk 2 above)
Comment (remya-james) [reply definitions/private-law/json/AuthorisationCaseField/CaseFileView/AuthorisationCaseField.json:169]: Done

**Pull request closed without merging**